File: atv-topico-3-4/topico-4-js/swap.js
const swap = (a, b, c) => {
  if (a.length > b && a.length > c) {
    let aux = a[b];
    a[b] = a[c];
    a[c] = aux;
  }
  console.log(a);
};

swap([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 0, 9);

File: atv-topico-3-4/topico-4/swap.hs
swap d e f = antes ++ [elemA] ++ meio ++ [elemB] ++ final
  where
    antes = take e d 
    elemA = d !! f 
    meio = drop (e+1) (take f d)
    elemB = d !! e
    final = drop (f+1) d


main = do
    a <- readLn :: IO [Int]
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    print $ swap a b c